services/app-web/src/components/Form/FieldYesNo/FieldYesNo.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { useField } from 'formik'
+import { Fieldset } from '@trussworks/react-uswds'
+import { FieldRadio } from '../FieldRadio/FieldRadio'
+import { PoliteErrorMessage } from '../../'
+
+import styles from './FieldYesNo.module.scss'
+
+/**
+ * This component renders a pair of radio buttons with Yes and No as the options.
+ *
+ * It relies on the Formik useField hook to work, so it must ALWAYS be rendered
+ * inside of a Formik form context.
+ *
+ * If you want to use these components outside a Formik form, you can use the
+ * ReactUSWDS components directly.
+ */
+
+export type FieldYesNoProps = {
+    name: string
+    label: string
+    showError: boolean
+    id: string
+} & JSX.IntrinsicElements['input']
+
+export const FieldYesNo = ({
+    name,
+    label,
+    showError,
+    id,
+    ...inputProps
+}: FieldYesNoProps): React.ReactElement => {
+    const [_field, meta] = useField({ name })
+
+    const isRequired =
+        inputProps['aria-required'] !== false && inputProps.required !== false // consumer must explicitly say this field is not required, otherwise we assume aria-required
+    return (
+        <Fieldset
```

Comment:
Yes, this looks like the right use of this element

Reply:
🔴  alright, lets remove the fieldset at this level.  

From webaim
> Fieldset and legend should only be used when a higher-level label is necessary. Single checkboxes or basic radio buttons that make sense from their labels alone do not require fieldset and legend. Nested fieldsets can cause odd screen reader behavior and should be avoided

We aren't using a higher level label here anyway and these are just basic two option radios.

Reply:
I added the legend back and it's working. The docs do say it's not ideal to nest field sets but our screen reader is working right now